src/main/resources/db/migration/V0_0_1__init.sql

```diff
@@ -1,4 +1,4 @@
- CREATE TABLE IF NOT EXISTS project
+CREATE TABLE IF NOT EXISTS project
```

Comment:
did you even try this? you cannot change migration files as the checksum will not be the same "Validate failed: Migration checksum mismatch for migration version 0.0.1"
should just change the new migration in this feature, which was not applied by everyone using develop-branch